packages/nimble-components/CONTRIBUTING.md

```diff
@@ -131,44 +131,97 @@ Use the `html` tagged template helper to define your custom template. See [Decla
 
 This package follows the [NI JavaScript and TypeScript Styleguide](https://github.com/ni/javascript-styleguide) with some exceptions listed in [Coding Conventions](/packages/nimble-components/docs/coding-conventions.md).
 
-#### API naming
+#### CSS
 
-Use lower-kebab-case for attributes and enum values that are part of a component's public API.
+Component CSS should follow the patterns described in [CSS Guidelines](/packages/nimble-components/docs/css-guidelines.md).
 
-```ts
-    @attr({ attribute: 'error-text' })
-    public errorText: string | undefined;
-```
+#### Represent control states as attributes
 
-#### CSS
+##### Why attributes over classes
```

Comment:
The "Responding to attribute values" section captures my primary concern in the form of best practices. Explicitly my concern was that over-using attributes would lead us to not strongly consider style only solutions and cause more JS behavior solutions with less optimal performance. That section better spells-out the tradeoffs and as long as we make best effort attempts to start with style-based behavior for attribute values I think my primary concern is mitigated.

Reply:
Let me know if there should be another way to capture that unless this comment thread in the PR is good enough.